docker/ckan/files/scripts/entrypoint.sh

```diff
@@ -1,18 +1,21 @@
 #!/bin/bash -e
 
-# load env vars from ${APP_DIR}/.env
-set -o allexport
-. ${APP_DIR}/.env
-set +o allexport
+echo "Executing entrypoint.sh ($ENV_NAME)"
 
-echo "Executing entrypoint.sh ($IS_DEV_ENV)"
+source ${APP_DIR}/venv/bin/activate
 
-if [ "$IS_DEV_ENV" = "true" ] ; then
+if [ $ENV_NAME == "local" ] ; then
```

Comment:
Creo que aca seria un solo `=`
```suggestion
if [ $ENV_NAME = "local" ] ; then
```